lib/agent/reports/actions.js

```diff
@@ -2,4 +2,8 @@ exports.actions = [
     'fullwipewindows',
     //'wipe',
     'factoryreset'
-  ];
\ No newline at end of file
+  ];
+
+  exports.actions_not_allowed = [
```

Comment:
user_activated is not part of the reports, so this array should not be here. maybe an array in command.js will suffice as the validation will be executed within that file